pkg/sink/sink.go

```diff
@@ -92,6 +93,8 @@ func (r Sink) HandleEvent(response http.ResponseWriter, request *http.Request) {
 				interceptor = github.NewInterceptor(t.Interceptor.Github, r.KubeClientSet, r.EventListenerNamespace, log)
 			case t.Interceptor.Gitlab != nil:
 				interceptor = gitlab.NewInterceptor(t.Interceptor.Gitlab, r.KubeClientSet, r.EventListenerNamespace, log)
+			case t.Interceptor.CEL != nil:
```

Comment:
Can we add a test at the interceptor level to make sure things work as expected end to end?